charts/corda/templates/bootstrap-cluster-db-secret.yaml

```
@@ -0,0 +1,19 @@
+{{- if or ( .Values.bootstrap.db.cluster.password.value) (.Values.bootstrap.db.cluster.username.value) }}

```

nit: you don't really need the parentheses here given they're only single values - you do have them then I tend to be more generous with the whitespace e.g. ` ( .Values.foo ) `
```suggestion
{{- if or .Values.bootstrap.db.cluster.password.value .Values.bootstrap.db.cluster.username.value }}

```